### ISSUE TITLE
the reconstruction is poor than original 3d GS with my own dataset

### ISSUE THREAD

**Issue by cuijianzhu** [ubc-vision/3dgs-mcmc#28]
when I run the code,Evaluating test PSNR is smaller than Evaluating train PSNR. And the reconstruction is poor than original 3d GS.
![952f9cd343492ed2a2bd4c27be4f4e8](https://github.com/user-attachments/assets/4642cbef-cc21-4035-b58d-c77d33bd23ca)

Optimizing 
Output folder: ./output/15222c0b-a [09/12 16:15:24]
Tensorboard not available: not logging progress [09/12 16:15:24]
Reading camera 36/36 [09/12 16:15:24]
Generating random point cloud (100000)... [09/12 16:15:24]
Loading Training Cameras [09/12 16:15:24]
Loading Test Cameras [09/12 16:15:25]
Number of points at initialisation :  100000 [09/12 16:15:25]
Training progress:  23%|▋  | 7000/30000 [01:14<13:35, 28.19it/s, Loss=0.0236660]
[ITER 7000] Evaluating test: L1 0.042469036579132084 PSNR 20.44259490966797 [09/12 16:16:40]

[ITER 7000] Evaluating train: L1 0.016292887553572655 PSNR 30.114678192138673 [09/12 16:16:40]

[ITER 7000] Saving Gaussians [09/12 16:16:40]
Training progress: 100%|██| 30000/30000 [43:15<00:00, 11.56it/s, Loss=0.0037885]

[ITER 30000] Evaluating test: L1 0.051942346990108496 PSNR 18.60160903930664 [09/12 16:58:41]

[ITER 30000] Evaluating train: L1 0.003593580843880773 PSNR 44.11332321166992 [09/12 16:58:41]

[ITER 30000] Saving Gaussians [09/12 16:58:41]

Training complete. [09/12 16:58:57]

**Comment by whwh747** [comment]
it looks bad

**Comment by shakibakh** [comment]
@cuijianzhu Training PSNR is expected to be better than test PSNR. Your final train PSNR of 44 suggests that maybe you are overfitting the training data. I suggest that you log the test loss (It's already part of the code if you install tensorboard), and see if the test loss is decreasing or remains high throughout the training.

**Comment by cuijianzhu** [comment]
> @cuijianzhu Training PSNR is expected to be better than test PSNR. Your final train PSNR of 44 suggests that maybe you are overfitting the training data. I suggest that you log the test loss (It's already part of the code if you install tensorboard), and see if the test loss is decreasing or remains high throughout the training.

![2024-12-25 19-15-24屏幕截图](https://github.com/user-attachments/assets/fed29fe9-7e05-45f0-9ceb-9a365eda2e0e)

**Comment by cmh1027** [comment]
@shakibakh Such overfitting is also observed in other scenes like _playroom_ of Deep Blending. Any ideas of a reason for this phenomenon?
